lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -35,20 +38,21 @@ def check_output2(*popenargs, **kwargs):
     if 'input' in kwargs:
         if 'stdin' in kwargs:
             raise ValueError('stdin and input arguments may not both be used.')
-        inputdata = kwargs['input']
+        inputdata = to_bytes(kwargs['input'])
```

Comment:
Also prepend inputdata with ```b_``` so everyone knows it's a variable intended to hold a byte value.